base/linalg/schur.jl

```diff
@@ -111,8 +111,7 @@ schur(A::Symmetric) = schur(full(A))
 schur(A::Hermitian) = schur(full(A))
 schur(A::UpperTriangular) = schur(full(A))
 schur(A::LowerTriangular) = schur(full(A))
-
-
+schur(A::Tridiagonal) = schur(full(A))
```

Comment:
These methods can obviously be made vastly more efficient.  For example, an upper-triangular matrix is *already* in Schur form.  But that can be left for another PR.